crates/fluvio-cluster/src/start/common.rs

```diff
@@ -20,6 +23,18 @@ static MAX_SC_LOOP: Lazy<u8> = Lazy::new(|| {
     var_value.parse().unwrap_or(120)
 });
 
+#[derive(Debug)]
+enum TryConnectError {
+    Timeout,
+    #[allow(dead_code)]
```

Comment:
why dead code?

Reply:
`rustc` doesn't consider debug prints as "read".
it says:
`warning: fields `expected` and `current` are never read`

Reply:
hmm.  I guess ok